google/cloud/pubsub_v1/types.py

```diff
@@ -112,6 +114,10 @@
         "The maximum amount of time in seconds to hold a lease on a message "
         "before dropping it from the lease management."
     )
+    FlowControl.max_extension_period.__doc__ = (
+        "Bounds the maximum amount of time in seconds before a message redelivery "
```

Comment:
Maybe start with:
"The max amount of time for a single lease extension/renewal attempt."